alcs-frontend/src/app/features/notice-of-intent/proposal/preparation/preparation.component.ts

```diff
@@ -59,75 +59,24 @@ export class PreparationComponent implements OnInit {
 
   private async loadSubtypes() {
     if (this._noticeOfIntent) {
-      const submission = await this.noticeOfIntentSubmissionService.fetchSubmission(this._noticeOfIntent.fileNumber);
-
       const subtypes = await this.noticeOfIntentService.listSubtypes();
       this.subtypes = subtypes;
 
       this.selectableTypes = subtypes.map((subtype) => {
-        let disabled = false;
-
-        disabled = this.isSubtypeDisabled(submission, subtype);
-
         return {
           label: subtype.label,
           value: subtype.label,
-          disabled,
+          disabled: false,
```

Comment:
I think disabled is optional parameter so you could simply ignore it here and it should act as false

Reply:
Yeah, it wasn't for `selectableTypes`, but this was just a placeholder type to pass to the dropdown where it is option, so I just removed the flag completely from this component.